tests/layer_tests/tensorflow_tests/test_tf_Bucketize.py

```diff
@@ -4,90 +4,43 @@
 import numpy as np
 import pytest
 import tensorflow as tf
-from common.layer_test_class import check_ir_version
 from common.tf_layer_test_class import CommonTFLayerTest
 
-from unit_tests.utils.graph import build_graph
-
 
 class TestBucketize(CommonTFLayerTest):
-    def create_bucketize_net(self, input_shape, input_type, boundaries_size, ir_version,
-                             use_new_frontend):
-        """
```

Comment:
I may ask to add additional comments with link on original operation, but could you leave such comments in place where it already exist?

Reply:
I would not like to add additional comment with a link to the operation because it is easy to find.